### PR TITLE
16 create functionality to recognize user input ie mouse clicks

### LINKED ISSUE #16: Create functionality to recognize user input i.e mouse clicks
**Points: 5**

RESIZED: 13

Tests: logicsim/tests.py

### PR DESCRIPTION
**Pull request by Fahim-A-Karim**
First iteration of button

### CODE REVIEW

**Review by Nguyen-Tommy — APPROVED** [review]
Didn't find problems.

**Review by Electrimatics — DISMISSED** [review]
Ticket #14 has "logicsim" and this PR has "logicSim".  This is causing issues with git and switching between branches

File: logicSim/button.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (joshuas4) [inline logicSim/button.html]: Duplicate button.html?

File: logicSim/views.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from django.shortcuts import render
```
Comment (joshuas4) [inline logicSim/views.py]: Duplicate import lines- delete one

**Comment by Fahim-A-Karim** [comment]
Implemented the ability to click something on the user interface when running our django project. This was just the html implementation of the button and in the future this should be able to work with other features but it doesn't currently.

**Review by joshuas4 — APPROVED** [review]

**Review by Electrimatics — APPROVED** [review]
Comment addressed.  This feature now displays an interactive button.  When you click on it, it will display the date.  To view the button, change logicsim/views.py line 14 to render 'logicsim/buttom.html'.  This is intended, since it is just a proof of concept.  We will be integrating buttons into the produce next iteration.

**Pull request merged**